pkg/scaler/machineSetScaler.go

```diff
@@ -19,14 +19,17 @@ import (
 )
 
 const (
-	LABEL_UPGRADE         = "upgrade.managed.openshift.io"
-	LABEL_MACHINESET      = "machine.openshift.io/cluster-api-machineset"
+	// LABEL_UPGRADE is the label used for managed upgrades
+	LABEL_UPGRADE = "upgrade.managed.openshift.io"
+	// LABEL_MACHINESET is the label used for machine sets
```

Comment:
Since a `machine` and `machineset` are separate resources:
`s/machine sets/machinesets/`

Reply:
resolved with https://github.com/openshift/managed-upgrade-operator/pull/242/commits/553786a65d79a58b6fdf46d29dd554795643730d